slsim/Sources/QuasarCatalog/number_density_quasar.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.interpolate import interp1d
+from astropy.table import Table
+from matplotlib.lines import Line2D
+
+""" 
+References: 
+Richards et al. 2005: DOI: 10.1111/j.1365-2966.2005.09096.x 
```

Comment:
can you put the references directly in the specific definitions?

Reply:
What do you mean by a specific definition? Do you mean you want a citation like you would do in an actual paper?

Reply:
definitions = functions, just ad the DOI links to the document strings in the functions